src/libs/error.mock.ts

```diff
@@ -9,13 +9,23 @@ import _ from 'lodash/fp';
  */
 export const errorWatcher = jest.fn();
 
+export const mockWithErrorReporting = _.curry((title, fn) => {
```

Comment:
Mocking `reportError` is not straightforward because ImportWorkflow uses it via `withErrorReporting`. Thus, it will still use the real `reportError` even if `reportError` is mocked like so:
```ts
jest.mock('src/libs/error', () => ({
  ...jest.requireActual('src/libs/error'),
  reportError: jest.fn(),
```

To work around this, we have to mock `withErrorReporting`. This extends the mock error module created for IA tests.